csharp-basics/exercises/Loops/Loops/Exercise1/Program.cs

```diff
@@ -4,19 +4,16 @@ namespace Exercise1
 {
     class Program
     {
+        //todo - write a program in C# to display the first 10 natural numbers
```

Comment:
Ja gribi atstāt kometāru, tad raksti, The program displays the first 10 natural numbers. Lai var saprast, ka tas jau ir pabeigts uzd

Reply:
Done